zad6/app/controllers/products_controller.rb

```diff
@@ -0,0 +1,12 @@
+class ProductsController < ApplicationController  
+  def index
+  	@customer ||= Customer.find_by!(id: params[:customer_id])
+  	@products = @customer.products.sorted.higher_than(price_param)
```

Comment:
Wg treści miały być produkty tańsze niż podane w parametrze `price`, ale to nie byłaby duża zmiana 😉